assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -411,4 +410,74 @@ describe('ExecutionResults', () => {
       )
     ).toBeTruthy();
   });
+
+  it('should open remediation modal when clicking on checkID', async () => {
+    const {
+      clusterID,
+      hostnames,
+      loading,
+      catalog,
+      executionError,
+      executionStarted,
+      executionResult,
+      checks,
+    } = prepareStateData('completed');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionLoading={loading}
+        executionStarted={executionStarted}
+        executionData={executionResult}
+        executionError={executionError}
+        clusterSelectedChecks={checks}
+      />
+    );
+
+    const { id: checkID, remediation } = catalog[0];
+    const clickableID = screen.getByText(checkID);
+    expect(clickableID.textContent).toBe(checks[0]);
+    let remediationModalElement = screen.queryByText(remediation);
+    expect(remediationModalElement).not.toBeInTheDocument();
+    fireEvent.click(clickableID);
+    remediationModalElement = screen.getByText(remediation);
+    expect(remediationModalElement).toBeInTheDocument();
+  });
+
+  it('should not open remediation modal when clicking on description', async () => {
+    const {
+      clusterID,
+      hostnames,
+      loading,
+      catalog,
+      executionError,
+      executionStarted,
+      executionResult,
+      checks,
+    } = prepareStateData('completed');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionLoading={loading}
+        executionStarted={executionStarted}
+        executionData={executionResult}
+        executionError={executionError}
+        clusterSelectedChecks={checks}
+      />
+    );
+
+    const { remediation, description } = catalog[0];
+    const checkDescriptionText = screen.getByText(description);
+    expect(description).toBe(checkDescriptionText.textContent);
```

Comment:
This looks inverted.
`expect(checkDescriptionText.textContent).toBe(description)`